src/components/protocol_handler/src/protocol_payload.cc

```diff
@@ -93,6 +93,7 @@ std::ostream& operator<<(std::ostream& os, const ProtocolPayloadV2& payload) {
             << ", data (bytes): " << payload.data.size() << ")";
 }
 
+// cppcheck-suppress unusedFunction
 size_t ProtocolPayloadV2SizeBits() {
```

Comment:
@Ypostolov It seems like this function is unused. Please, check it on the both flow and remove if it is unused.